kuma/users/views.py

```diff
@@ -745,6 +767,38 @@ def recover(request, uidb64=None, token=None):
     return render(request, 'users/recover_failed.html')
 
 
+@require_POST
+def create_stripe_subscription(request):
+    user = request.user
+
```

Comment:
Atm I don't see why that'd be necessary. Is the concern UX? Because the only wrong thing the user can do here is f-up the email address, which stripe takes care of validating in its form. I don't think we need UX for users who craft their own requests 🙃 
Is the concern security? Cause I don't see an angle here.
Or is it that we might get logs from users who send invalid values? If we are worried about that one, I could add another catch-block for invalid token/email.

Reply:
It's just really good practice to protect your view function code with some validation. 
Don't confuse Django forms for just rendering `<input type....` template strings. It's also a really good framework for validating and transforming `request.GET` (or `request.POST`) to clean data before doing anything with it.

Reply:
I do agree about the value of django forms in general, but I don't see the value it would provide in this case.